src/index.js

```diff
@@ -0,0 +1,23 @@
+import './styles/h5p-components.css';
+import CoverPage from './components/h5p-cover-page';
+import Button from './components/h5p-button';
+import Draggable from './components/h5p-draggable';
+import Dropzone from './components/h5p-dropzone';
+import Navigation from './components/h5p-navigation';
+import PlaceholderImg from './components/h5p-placeholder-img';
+import ProgressBar from './components/h5p-progress-bar';
+import ProgressDots from './components/h5p-progress-dots';
+import ResultScreen from './components/h5p-result-screen';
```

Comment:
Import paths of files should end with `.js` (see eslint). It makes it clearer that this is not an external module that's imported (and some webpack extensions would even require it to work).